django/contrib/admin/static/admin/img/icon-debug.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="13" height="13" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
+  <path fill="#adadad" d="M896 1590.4c112 0 201.6-37.33 283.73-119.47 74.67-74.67 119.47-171.73 119.47-283.73v-395.73c0-112-37.33-201.6-119.47-283.73-74.67-74.67-171.73-119.47-283.73-119.47s-201.6 52.27-283.73 126.93c-82.13 74.67-119.47 171.73-119.47 283.73v395.73c0 112 37.33 201.6 119.47 283.73 82.13 82.13 171.73 112 283.73 112ZM694.4 1291.73h395.73v-201.6h-395.73v201.6ZM694.4 896h395.73v-201.6h-395.73v201.6ZM896 1792c-104.53 0-209.07-29.87-298.67-82.13-89.6-52.27-164.27-126.93-216.53-216.53H97.07v-201.6h209.07c0-29.87-7.47-67.2-7.47-97.07v-97.07H97.07v-201.6h201.6v-97.07c0-29.87 7.47-67.2 7.47-97.07H97.07v-201.6h276.27c22.4-37.33 52.27-74.67 74.67-104.53 37.33-37.33 74.67-67.2 112-89.6l-164.27-164.27L537.6 0l216.53 216.53c44.8-14.93 97.07-22.4 141.87-22.4s97.07 7.47 141.87 22.4L1254.4 0l141.87 141.87-164.27 164.27c37.33 22.4 74.67 52.27 104.53 89.6 29.87 37.33 59.73 67.2 82.13 104.53h276.27v201.6h-209.07c7.47 29.87 7.47 67.2 7.47 97.07v97.07h201.6v201.6h-201.6v97.07c0 29.87 0 67.2-7.47 97.07h209.07v201.6h-276.27c-52.27 89.6-126.93 164.27-216.53 216.53-97.07 52.27-201.6 82.13-306.13 82.13z"/>
```

Comment:
The debug icon contrast in light mode is not strong enough

Reply:
It might be hard to get the colors with enough contrast for the icons, I tried to match to proportions to fit with other messages, but even currently implemented ones do not really meet the requirements.

The color has to be good enough for both light and dark mode, as the icon color does not change. I can see changing it so it uses the CSS variable, which would allow control over the icon color and change it between the themes, maybe in another ticket?

![image](https://github.com/user-attachments/assets/23ce80ac-92a3-467a-bd8e-398d1a8114db)